spec/api/articles_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe 'GET /api/articles', type: :request do
+  subject { response }
+  before do
+    get '/api/articles'
+  end
```

Comment:
This `before` block will be executed before each `describe` block. This means that you will hit the server multiple times since you are making the requests in those blocks as well.